Makefile

```diff
@@ -21,6 +21,9 @@ clean:
 docs:
 	$(GRADLE) javadoc
 
+checkstyle:
+	$(GRADLE) checkstyleMain
```

Comment:
Could you add a [step to CI to run this](https://github.com/infinyon/fluvio-client-java/blob/f701913f20a28a07a459c110a30793ea0b95aac2/.github/workflows/ci.yml#L47-L49)

Reply:
Done